MovieGPT-api/src/main/java/lt/techin/group/project/model/Role.java

```diff
@@ -0,0 +1,24 @@
+package lt.techin.group.project.model;
+
+import jakarta.persistence.*;
+import lombok.*;
+
+import java.util.Set;
+
+
+@Setter
+@Getter
+@Entity
+public class Role {
```

Comment:
Just asking: would it not be better to have Role as enum? And is it really needed to make a List of our every user that has that role? This info is saved in DB near user - and can be easily fetched each time if needed, right? I just can't think of any cases when could it be needed

Reply:
As we only have 2 roles and it'll not change, enum sounds nice for it, I try to make it.
But why we have roles list in user I tried to explain here: https://github.com/Stalkakuma/MovieGPT/pull/2#discussion_r1874769840
I think same idea implement to front end, where you don't need to separate authority access for admin to basic user access, you just grant everything to user, and lock what admin can access.